Check and balances.md

```diff
@@ -11,11 +11,24 @@ In this case. Delijaj is the seller and Nyomi the buyer.
 While testing the trading system on the 3<sup>rd</sup> auction, a seller decided to trad some parts of the project. 
 However, she did not own any part of it but the website let her do it anyway. The parts were at 50 token. A buyer 
 decided to acquire those parts. The exchange was made and the seller finished with a -50 on the project (like a debt).
-This shouldn't happend since the parts must be only managed by the contributors and nobody else.
+This shouldn't happend since the parts must be only managed by the contributors and nobody else, and also to prevent any scam.
 
-## Sponsor/Inversor 
-As the website is today, the trade is at its basic stage. To improve it, we can put a system of sponsor/investor.
-Indeed if we look at the projects, some people will have fantastic ideas but not the means to do it. So to help developping 
-the market and the motivation to creat a projet. In the case that a person doesn't want to continue the project, for any reasons,
-she could sell it in the platforme to keep it going.
-It will creat a real market and viability of the projects.
+## Licence
+I don’t see the point on creating a new licence. I feel like we should use what we already have since creating such a licence 
+would be assembling existing licences. (and some project would have similar stuff). 
+“provisional financial attribution based on share ownership, which itself is based on the contribution & trading trails
+over the whole history of the project.” This is not in the good place for me, it should be shown at the contributor’s 
+profile and thus on how we manage the profiles.
+
+## Encouragement to Share
+In what way?By social media? By sharing with other people you “followed” on the platform?
+I don’t see how other than having a kind of friend list and we already have a way to advise people for projects.
+Since in the whitepaper there isn’t anything written, we could give some ideas.
+
+## Merge of two projects
+As in the whitepaper it is stated that they don’t know how to make this happened, I will make a proposition.
+I feel like this should be in place with a kind of peer to peer comms, in other words, having the contributors or 
+investors looking at projects and notify the person (with a button on the project or just by emailing the “chief” of the project).
+
```

Comment:
I don't see what you mean by that ? A powerful mechanism for something like that is to keep the original two projects so that contributor history is not lost and use them as functions in the new merged project... I am not sure how to implement this otherwise, Also that some projects will not be suitable to such a means... The main danger of merging in a new repo is the loss of contributions